test/e2e-tests.sh

```diff
@@ -39,13 +39,15 @@ export_logs_continuously "kafka-broker-dispatcher" "kafka-broker-receiver" "kafk
 
 failed=false
 
-go_test_e2e -timeout=30m ./test/... || failed=true
+go_test_e2e -timeout=30m ./test/e2e_new || failed=true
+
+go_test_e2e -timeout=30m ./test/e2e || failed=true
```

Comment:
We need 3 dots to be recursive so that it's much more robust, I mean in the future we might add subdirectories for whatever reason:
```suggestion
go_test_e2e -timeout=30m ./test/e2e_new/... || failed=true

go_test_e2e -timeout=30m ./test/e2e/... || failed=true
```